docs/library/sys.rst

```diff
@@ -69,14 +69,16 @@ Constants
    MicroPython, it has following attributes:
 
    * *name* - string "micropython"
-   * *version* - tuple (major, minor, micro), e.g. (1, 7, 0)
+   * *version* - tuple (major, minor, micro, releaselevel), e.g. (1, 22, 0, '')
    * *_machine* - string describing the underlying machine
    * *_mpy* - supported mpy file-format version (optional attribute)
 
    This object is the recommended way to distinguish MicroPython from other
    Python implementations (note that it still may not exist in the very
    minimal ports).
 
+   Starting with version 1.21.0, the 4th node *releaselevel* in *implementation.version* is either an empty string or *'preview'*
```

Comment:
I think that should say "Starting with version 1.22.0", because at the 1.21.0 release this fourth field was not present.